src/components/GameCard.tsx

```diff
@@ -9,17 +9,71 @@ interface GameCardProps {
     gameTitle: string;
     gameImage: any;
     gameDescription: string;
+    gamePlatforms: Array<string>;
+    gameGenres: Array<string>;
+    gameUniverses: Array<string>;
+    gameCompanies: Array<string>;
+    gameReleaseDate: Array<string>;
+    gameRating?: number;
+    gameReview?: string;
+    gameRanking?: string;
+    yearPlayed?: number;
+    yearReceived?: number;
+    notes?: string;
+    platformsOwnedOn?: Array<string>;
+    dateAdded?: String;
+
 }
 
-export function GameCard({gameTitle, gameImage, gameDescription } : GameCardProps ) {
+export function GameCard(gameCardProps : GameCardProps ) {
     // will eventually be an API function
     const addToLibrary = () => true;
 
     return (
         <div>        
-            <h1>{gameTitle}</h1>
-            <svg>{gameImage}</svg>
-            <p>{gameDescription}</p>
+            <h1>{gameCardProps.gameTitle}</h1>
+            <img 
+            src = {gameCardProps.gameImage[0]}
+            alt = "Game"
+            />
+            <p>{gameCardProps.gameDescription}</p>
+            <div>
+                {(gameCardProps.gamePlatforms.map((platform) => {
+                    return <p>{platform}</p>
+                }))}
+            </div>
+            <div>
+                {(gameCardProps.gameGenres.map((genre) => {
+                    return <p>{genre}</p>
+                }))}
+            </div>
+            <div>
+                {(gameCardProps.gameUniverses.map((universe) => {
+                    return <p>{universe}</p>
+                }))}
+            </div>
+            <div>
+                {(gameCardProps.gameCompanies.map((company) => {
+                    return <p>{company}</p>
+                }))}
+            </div>
+            <div>
+                {(gameCardProps.gameReleaseDate.map((date) => {
+                    return <p>{date}</p>
+                }))}
+            </div>
+            <p>{gameCardProps?.gameRating}</p>
+            <p>{gameCardProps?.gameReview}</p>
+            <p>{gameCardProps?.gameRanking}</p>
+            <p>{gameCardProps?.yearPlayed}</p>
+            <p>{gameCardProps?.yearReceived}</p>
+            <p>{gameCardProps?.notes}</p>
```

Comment:
Slight nitpick but should notes be at the end? Since that'll probably be longer than the other fields

In general this ordering might need a designer's input if we can get one

Reply:
That or review, most likely. I'm not going to change it this PR since I think this would be something we'd ultimately change the whole page view based on a design, but that is a good point